### PR TITLE
use delete range by default

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

File: tests/config/mod.rs [hunk 1]
```diff
@@ -124,7 +124,7 @@ fn test_serde_custom_tikv_config() {
         raft_store_max_leader_lease: ReadableDuration::secs(12),
         right_derive_when_split: false,
         allow_remove_leader: true,
-        use_delete_range: true,
+        use_delete_range: false,
```
Comment (siddontang) [inline tests/config/mod.rs]: true is ok

**Comment by siddontang** [comment]
Rest LGTM

File: tests/config/mod.rs (hunk 1 above)
Comment (zhangjinpeng1987) [reply tests/config/mod.rs]: In config test, we prefer don't use default value.

**Comment by huachaohuang** [comment]
CI fails because of https://github.com/facebook/rocksdb/issues/3592.

**Comment by siddontang** [comment]
any update? Can we figure out a way to get around it? We need publishing this feature ASAP.

@zhangjinpeng1987 @huachaohuang

**Comment by huachaohuang** [comment]
Sent a PR: https://github.com/facebook/rocksdb/pull/3614

**Comment by siddontang** [comment]
LGTM
PTAL @huachaohuang

**Comment by siddontang** [comment]
CI failed @zhangjinpeng1987

**Pull request merged**